src/translations/translationHelper.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
+
+const isRootLocaleComment =
+  reply => (reply.depth === 1 && reply.body.length === 4 && reply.body.substring(0, 2) === '# ');
+
+export const toObject = (body) => {
+  const object = {};
+  const lines = body.split(/\n/);
+  lines.forEach((line) => {
+    const message = line.split(':');
```

Comment:
This code will have unwanted results if the translation strings includes `:` inside like this :
```json
busy_team: 'We are proud to have: Fabien, Pooria, etc'
```